requirements.txt

```diff
@@ -1,5 +1,10 @@
-django
-django-environ
-psycopg2
-whitenoise
-django-ckeditor
\ No newline at end of file
+asgiref==3.8.1
+Django==5.0.4
+django-allauth==0.62.0
+django-ckeditor==6.7.1
+django-ckeditor5==0.0.4
```

Comment:
change django-ckeditor5==0.0.4 to django-ckeditor-5==0.2.12
remove django-ckeditor==6.7.1